f/medical/fn_medicalPreInit.sqf

```diff
@@ -0,0 +1,61 @@
+// F3 - Medical Systems Pre Initialisation
+// Credits: Please see the F3 online manual (http://www.ferstaberinde.com/f3/en/)
+// ====================================================================================
+
+// Wait until medical param is initialized by the pre init param processor
+// Pre Init is unscheduled space...
+//waitUntil
+//{
+//	!(isNil {missionNamespace getVariable "f_var_medical"})
+//};
```

Comment:
It will require some testing but I believe there is a good chance we make this happen!